### PR TITLE
Added ime option

### LINKED ISSUE #407: Set ime option to edit text in login activity
**Describe the bug**
Click on keyboard enter or next button it is expanding the edit text rather then pointing directly to next edit text.

**To Reproduce**
1.Open login activity.
2.  enter user name.
3. click on keyboard next button.
Not pointing to next edit text rather it expanding the edit text of username.
<table>
<th>Gif of the current behaviour</th>
<tr>
<td>
</td>
</tr>
</table>

**Why we need this**
For better user experience click on next in keyboard should point to next edit text.

### PR DESCRIPTION
**Pull request by arvindnegi1**
### Description
Fixes not pointing to next edit text on keyboard ime action clicked.
Fixes not submit implicitly on password edit text ime option.

Fixes #407 
**What changes are done**
- Set ime option to edit text in sign in layout
- Set editor action for password ime option clicked.

### CODE REVIEW

**Comment by arvindnegi1** [comment]
@dewall sir, please review the changes, thankyou :)

File: org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java [hunk 1]
```diff
@@ -115,6 +116,15 @@ protected void onSwitchToRegisterClicked() {
         SignupActivity.startActivity(this);
     }
 
+    @OnEditorAction(R.id.activity_login_password_input)
+    protected boolean implicitSubmit()  {
+
```
Comment (dewall) [inline org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java]: remove empty line

File: org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java [hunk 2]
```diff
@@ -115,6 +116,15 @@ protected void onSwitchToRegisterClicked() {
...
+        View logInSubmit ;
+        logInSubmit=    findViewById(R.id.activity_signin_login_button);
```
Comment (dewall) [inline org.envirocar.app/src/org/envirocar/app/views/login/SigninActivity.java]: you need to autoformat

**Comment by arvindnegi1** [comment]
@dewall sir, please review . changes done.

**Pull request merged**